packages/BackToTopButton/index.vue

```diff
@@ -16,25 +16,45 @@
 <script>
 export default {
   name: 'BackToTopButton',
+  props: {
+    targetEl: {
+      type: [Element, Window],
+      default: null
```

Comment:
If it's not required, every usage should have `if (this.targetEl)`. E.g., when `destroyed`, `handleScroll`